app/script/view_model/content/InputBarViewModel.js

```diff
@@ -75,28 +76,42 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
 
     this.hasFocus = ko.pureComputed(() => this.isEditing() || this.conversationHasFocus()).extend({notify: 'always'});
     this.hasTextInput = ko.pureComputed(() => {
-      return this.conversationEntity() ? this.conversationEntity().input().length > 0 : false;
+      return this.conversationEntity() ? this.conversationEntity().input().text.length > 0 : false;
+    });
+
+    this.inputTextOnly = ko.pureComputed({
+      read: () => this.input().text,
+      write: value => this.input(value),
     });
 
     this.input = ko.pureComputed({
       read: () => {
+        let inputObject = {mentions: [], text: ''};
+
         if (this.isEditing()) {
-          return this.editInput();
-        }
+          inputObject = this.editInput();
+        } else if (this.conversationEntity() && this.conversationEntity().input()) {
+          inputObject = this.conversationEntity().input();
 
-        if (this.conversationEntity()) {
-          return this.conversationEntity().input() || '';
+          this.currentMentions = inputObject.mentions;
```

Comment:
Let's talk 1:1 to find a proper solution to completely get rid of this duplication of information